CHANGES.rst

```diff
@@ -6,6 +6,8 @@ Changes
 1.8.2 (unreleased)
 ------------------
 
+- :func:`show_ref` and :func:`show_backref` now accept a file-like object as an
+  alternative to a filename.
 
```

Comment:
I prefer two blank lines between version sections in the changelog.

If I ever decide to make the PyPI front page shorter that'll allow me to limit the changelog section to the latest few entries by splitting on `'\n\n\n'`.